crates/bootstrap_srv/src/http.rs

```diff
@@ -152,6 +152,19 @@ fn tokio_thread(
                 Some(sbd_server::CSlot::new(sbd_config.clone(), ip_rate.clone()))
             };
 
+            let rustls_config = if let Some(tls_config) = server_config.tls_config {
+                let rustls_config = tls_config
+                    .create_tls_config()
+                    .await
+                    .expect("Failed to create TLS config");
+
+                tokio::task::spawn(tls_config.reload_task(rustls_config.clone()));
```

Comment:
Capture abort handle and shut down the task in the server shutdown logic.

Reply:
Do'h, of course :) Done!